model/team_test.go

```diff
@@ -6,63 +6,56 @@ package model
 import (
 	"strings"
 	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
 )
 
 func TestTeamJson(t *testing.T) {
 	o := Team{Id: NewId(), DisplayName: NewId()}
 	json := o.ToJson()
 	ro := TeamFromJson(strings.NewReader(json))
 
-	if o.Id != ro.Id {
-		t.Fatal("Ids do not match")
-	}
+	require.Equal(t, o.Id, ro.Id, "Ids do not match")
 }
 
 func TestTeamIsValid(t *testing.T) {
 	o := Team{}
 
-	if err := o.IsValid(); err == nil {
-		t.Fatal("should be invalid")
-	}
+	err := o.IsValid()
+	require.NotNil(t, err, "should be invalid")
 
 	o.Id = NewId()
-	if err := o.IsValid(); err == nil {
-		t.Fatal("should be invalid")
-	}
+	err = o.IsValid()
+	require.NotNil(t, err, "should be invalid")
 
 	o.CreateAt = GetMillis()
-	if err := o.IsValid(); err == nil {
-		t.Fatal("should be invalid")
-	}
+	err = o.IsValid()
+	require.NotNil(t, err, "should be invalid")
 
 	o.UpdateAt = GetMillis()
-	if err := o.IsValid(); err == nil {
-		t.Fatal("should be invalid")
-	}
+	err = o.IsValid()
+	require.NotNil(t, err, "should be invalid")
 
 	o.Email = strings.Repeat("01234567890", 20)
-	if err := o.IsValid(); err == nil {
-		t.Fatal("should be invalid")
-	}
+	err = o.IsValid()
+	require.NotNil(t, err, "should be invalid")
 
 	o.Email = "corey+test@hulen.com"
 	o.DisplayName = strings.Repeat("01234567890", 20)
-	if err := o.IsValid(); err == nil {
-		t.Fatal("should be invalid")
-	}
+	err = o.IsValid()
+	require.NotNil(t, err, "should be invalid")
 
 	o.DisplayName = "1234"
 	o.Name = "ZZZZZZZ"
-	if err := o.IsValid(); err == nil {
-		t.Fatal("should be invalid")
-	}
+	err = o.IsValid()
+	require.NotNil(t, err, "should be invalid")
 
 	o.Name = "zzzzz"
 	o.Type = TEAM_OPEN
 	o.InviteId = NewId()
-	if err := o.IsValid(); err != nil {
-		t.Fatal(err)
-	}
+	err = o.IsValid()
+	require.Nil(t, err, err)
```

Comment:
nit: why display the error twice?

Reply:
should this be `require.Nil(t, err, err.Error())` instead?

Reply:
the previous logic used `err` as a message so I used the same logic